### PR TITLE
DWR-1092  Change getEffectiveDate to adhere to the Data Model format …

### PR DESCRIPTION
**Pull request by agorina**
…(which is 'YYYY-MM-DD').

### CODE REVIEW

File: model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java [hunk 1]
```diff
@@ -881,15 +883,15 @@ public void setAssessmentParticipantSessionPlatformUserAgent(String value) {
...
      * @return
      *     possible object is
-     *     {@link Date }
+     *     {@link LocalDate }
      *
      */
-    public Date getEffectiveDate() {
-        return effectiveDate;
+    public LocalDate getEffectiveDate() {
+        return effectiveDate.toInstant().atZone(ZoneId.of("GMT")).toLocalDate();
```
Comment (agorina) [inline model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java]: Is this what was the approach recommended in https://github.com/SmarterApp/RDW_Ingest/pull/376?

Comment (malaffoon) [reply model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java]: i want to dig around a little ... if `Date` has a toInstant() then i don't understand why we need to zone it to convert it to `LocalDate`

Comment (malaffoon) [reply model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java]: Did you try `LocalDate.from(effectiveDate.toInstant())`?

Comment (chdietz) [reply model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java]: But Instant is a point in time on the UTC timezone, correct?

Comment (agorina) [reply model/src/main/java/org/opentestsystem/rdw/common/model/trt/Opportunity.java]: Change it.

**Pull request merged**